diagnostic.md

```diff
@@ -31,51 +33,52 @@ What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```bash
-// your response here
+rails g scaffold things : generates a scaffold called things
+rails g controller things : generates a controller called things.
 ```
 
 What is the job of the `serializer`?
 
 ```bash
-// your response here
+to put assigned values into strings and on read creates them as ruby objects.
 ```
```

Comment:
Technically, what we're looking for here is that it filters out certain data from the user.